lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize(dishes)
+    @dishes = dishes
+  end
+
+def initialize
```

Comment:
Lint/DuplicateMethods: Method `Menu#initialize` is defined at both /app/lib/menu.rb:5 and /app/lib/menu.rb:9.